.github/workflows/pull_request_stats.yml

```diff
@@ -23,33 +23,34 @@ jobs:
         with:
           fetch-depth: 25
 
-      - run: echo ::set-output name=DOCS_CHANGE::$(node skip-docs-change.js echo 'not-docs-only-change')
+      - name: Check non-docs only change
+        run: echo ::set-output name=DOCS_CHANGE::$(node scripts/run-for-change.js --not --type docs --exec echo 'not-docs-only-change')
         id: docs-change
 
       - name: Setup node
         uses: actions/setup-node@v3
-        if: ${{ steps.docs-change.outputs.DOCS_CHANGE != 'docs only change' }}
+        if: ${{ steps.docs-change.outputs.DOCS_CHANGE == 'not-docs-only-change' }}
```

Comment:
Should we call this “code-change” or “change-includes-code”

Reply:
Updated the value of `DOCS_CHANGE` to be "no" when something else is modified, this should make it a bit more readable https://github.com/vercel/next.js/pull/38479/commits/aa2c280bbb7b1df701fa29dbb3fbd5baf6599b9a and matches the Azure check.